### ISSUE TITLE
[M0-Dev] Atributos de la clase Moneda

### ISSUE THREAD

**Issue by manujurado1** [ignaciotitos/IV---HealthForAll#12]
Persiguiendo el objetivo de la HU1 (#6) de saber cuanto valdrá su moneda en un futuro, me surgen problemas sobre como organizar dicha clase ya que desconozco el tema de como se suele medir la variación del valor de una moneda.

He planteado esta propuesta de la lista de atributos que debería tener una moneda:
- Nombre
- Valor actual
- Estimación valor dentro de 1 mes
- Estimación valor dentro de 1 año

De esta manera el usuario cumpliría su deseo y vería la estimación del valor de su moneda en 2 longitudes de tiempo diferentes.
Desconozco si se requieren más atributos o si los tiempos de estimación correctos deben ser otros, por ese motivo abro este issue.

**Comment by ignaciotitos** [comment]
sí, yo creo que la clase moneda con eso bastaría. Lo que sí que habría que crear es una estructura, un struct o algo en los que guardar los valores del PIB, Importaciones, Exportaciones, inflación...

**Comment by ignaciotitos** [comment]
Más que estimación dentro de un mes o dentro de un año, lo dejaría como estimación a secas, ya que puede guardar la estimación cuando sea.

**Comment by manujurado1** [comment]
Perfecto, queda claro la estructura que tendrá la clase moneda

**Comment by JJ** [comment]
"Atributos de la clase" no está formulado como un problema. Si ya presupones qué clases vas a tener, es imposible que modelices el problema para llegar a las clases necesarias.
